### PR TITLE
Fixes crash on large payloads for MGET/MSET/SCAN

### PR DESCRIPTION
**Pull request by akashdeepgoel**
Fixes the issue with payloads which exceed the size of a single MBUF causing dynomite server to crash.

We now return DYNOMITE_PAYLOAD_TOO_LARGE with an exception back to the client.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,12 @@
 language: c
 script: bash ./travis.sh
 
+before_install:
```
Comment (anujphadke) [inline .travis.yml:4]: Can you move this in a separate  commit as this is probably not related to the crash?

File: src/proto/dyn_redis.c [hunk 2]
```diff
@@ -3030,7 +3030,9 @@ static rstatus_t redis_copy_bulk(struct msg *dst, struct msg *src, bool log) {
   }
 
   p = mbuf->pos;
-  ASSERT(*p == '$');
+  if (*p != '$') {
```
Comment (anujphadke) [inline src/proto/dyn_redis.c:3033]: Not related to your change, but can p be NULL at 3032? Do you think it makes sense to check for that?

File: .travis.yml (hunk 1 above)
Comment (akashdeepgoel) [reply .travis.yml:4]: This is important for the automated travis build on github as this ensures correct pip version for the build

File: src/proto/dyn_redis.c (hunk 2 above)
Comment (akashdeepgoel) [reply src/proto/dyn_redis.c:3033]: All mbufs need to come from allocation code path which ensures mbuf->pos is set to the starting of the buffer. Ideally, this should never be NULL

**Review by pkarumanchi9 — APPROVED** [review]
changes look good.

**Review by anujphadke — APPROVED** [review]

**Pull request merged**